bip-musig2.mediawiki

```diff
@@ -402,9 +402,9 @@ Input:
 * Fail if ''d' = 0'' or ''d' &ge; n''
 * Let ''P = d'⋅G''
 * Let ''a = GetSessionKeyAggCoeff(session_ctx, P)''; fail if that fails
-* Let ''gp = 1'' if ''has_even_y(P)'', otherwise let ''gp = -1 mod n''
+* Let ''gpk = 1'' if ''has_even_y(P)'', otherwise let ''gpk = -1 mod n''
```

Comment:
It seems that `gp` should be updated in the refernece code as well, but #37 will remove `gp` anyway.